gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -161,6 +161,10 @@ static String encodeMetadataValues(byte[] bytes) {
     return bytes == null ? Data.NULL_STRING : BaseEncoding.base64().encode(bytes);
   }
 
+  // To track the http statistics
+  private HashMap<GoogleCloudStorageStatistics, AtomicLong> gcsStatistics =
+      new HashMap<GoogleCloudStorageStatistics, AtomicLong>();
+
```

Comment:
Looks like we are not using this var anywhere

Reply:
Updated